packages/commonwealth/client/scripts/views/pages/NotificationSettingsOld/SubscriptionEntry.tsx

```diff
@@ -0,0 +1,76 @@
+import NotificationSubscription from 'models/NotificationSubscription';
+import 'pages/notification_settings/index.scss';
+import React from 'react';
+import { useGetCommunityByIdQuery } from 'state/api/communities';
+import { CWCheckbox } from '../../components/component_kit/cw_checkbox';
+import { CWCommunityAvatar } from '../../components/component_kit/cw_community_avatar';
+import { CWText } from '../../components/component_kit/cw_text';
+import { CWToggle } from '../../components/component_kit/cw_toggle';
+
+type SubscriptionEntryProps = {
+  communitId: string;
+  subscriptions: NotificationSubscription[];
+  showSubscriptionsCount?: boolean;
+  canToggleEmailNotifications?: boolean;
+  areEmailNotificationsEnabled?: boolean;
+  onToggleReceiveEmailsNotifications: () => void;
+  canToggleInAppNotifications?: boolean;
+  areInAppNotificationsEnabled?: boolean;
+  onToggleReceiveInAppNotifications: () => void;
+};
+
+const SubscriptionEntry = ({
```

Comment:
good call for this but let's try to avoid additional refactoring in the future in such a big PRs